### PR TITLE
Remove engage and learn

### PR DESCRIPTION
**Pull request by kicferk1**
### Context
Now that Engage and Learn has its own repo, we should remove it from this one. 

### Changes proposed in this pull request
Burn everything with fire, leave nothing behind.
Remove models, policies, controllers, routes, css, specs, db tables, big chunk of seeding. Cut the connection between Core Induction Programme that LPs choose and Course Years.

### Guidance to review
I didn't bother making the migration dropping course tables revertible - if we really think it's important I can maybe make it happen, but then those tables will be pretty useless without the seeding data :/

### Testing
Don't think there is much to test here. People should not see any Core induction programme content in our review app I guess. The rest of the app should work.

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
 
+if git rev-parse --verify HEAD >/dev/null 2>&1
```
Comment (seb-siem-softwire) [inline .githooks/pre-commit:3]: A comment would be nice so that it's easier to read

File: .githooks/pre-commit [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+KEY_ID=$(git diff --cached --name-only -z $against | xargs -0 cat | perl -nle'print $& if m{(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])}')
+KEY=$(git diff --cached --name-only -z $against | xargs -0 cat | perl -nle'print $& if m{(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])}')
+
+if [ "$KEY_ID" != "" -o "$KEY" != "" ]; then
+    echo "Found patterns for AWS_ACCESS_KEY_ID/AWS_SECRET_ACCESS_KEY"
+    echo "Please check your code and remove API keys."
+    exit 1
+fi
+
+
```
Comment (seb-siem-softwire) [inline .githooks/pre-commit:24]: Could you zap out of existance one empty line ?

**Comment by seb-siem-softwire** [comment]
As long as Babu gives his blessing - I'm fine with it

**Review by amazimbe — COMMENTED** [review]
Feels like signing divorce papers 😢 .

File: .githooks/pre-commit [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+# Redirect output to stderr.
+exec 1>&2
+
+# Check changed files for an AWS keys
+KEY_ID=$(git diff --cached --name-only -z $against | xargs -0 cat | perl -nle'print $& if m{(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])}')
+KEY=$(git diff --cached --name-only -z $against | xargs -0 cat | perl -nle'print $& if m{(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])}')
+
+if [ "$KEY_ID" != "" -o "$KEY" != "" ]; then
+    echo "Found patterns for AWS_ACCESS_KEY_ID/AWS_SECRET_ACCESS_KEY"
+    echo "Please check your code and remove API keys."
```
Comment (ethanmills) [inline .githooks/pre-commit:20]: Do we have any of these?

**Review by rupollbabu — COMMENTED** [review]
material information for supplier missing. Requested kicferk1 to add supplier material.

**Comment by kicferk1** [comment]
> material information for supplier missing. Requested kicferk1 to add supplier material.

Works for me now :)

**Review by rupollbabu — APPROVED** [review]
Regression test completed, No impact on **service** after removing engage and learn.

**Comment by kicferk1** [comment]
> and nothing of value was lost

Alright, time to merge this bad boy.

**Pull request merged**